### PR TITLE
Add support for capitalize to text-transform

### PR DESCRIPTION
**Pull request by Tonkpils**
_Authors: Please fill out this form carefully and completely._

_Reviewers: By approving this Pull Request you are approving the code change, as well as its deployment and mitigation plans._
_Please read this description carefully. If you feel there is anything unclear or missing, please ask for updates._

### What are you trying to accomplish?

Add support for capitalize on text-transform. It's already supported by the CSS class `text-capitalize` https://primer.style/design/foundations/css-utilities/typography#typographic-styles

#### Risk Assessment

- [x] **Low risk** the change is small, highly observable, and easily rolled back.
- [ ] **Medium risk** changes that are isolated, reduced in scope or could impact few users. The change will not impact library availability.
- [ ] **High risk** changes are those that could impact customers and SLOs, low or no test coverage, low observability, or slow to rollback.

### What approach did you choose and why?

Added the change to the list where these utilities are already defined

### Anything you want to highlight for special attention from reviewers?

Is there anything else that needs to be updated?

### Merge checklist

- [x] Added/updated tests
- [x] Added/updated documentation
- [ ] Added/updated previews (Lookbook)
- [ ] Tested in Chrome
- [ ] Tested in Firefox
- [ ] Tested in Safari
- [ ] Tested in Edge

Take a look at the [What we look for in reviews](https://github.com/primer/react/blob/main/contributor-docs/CONTRIBUTING.md#what-we-look-for-in-reviews) section of the contributing guidelines for more information on how we review PRs.

### CODE REVIEW

File: lib/primer/classify/utilities.yml [hunk 1]
```diff
@@ -1558,6 +1558,8 @@
 :text_transform:
   :uppercase:
   - text-uppercase
+  :capitalize:
```
Comment (camertron) [inline lib/primer/classify/utilities.yml:1561]: This file is auto-generated, so this change will be overwritten the next time the utilities script runs. It gets populated by the contents of custom_utilities.yml, so the change below is all you need 👍

**Review by camertron — CHANGES_REQUESTED** [review]

**Review by keithamus — APPROVED** [review]

File: lib/primer/classify/utilities.yml (hunk 1 above)
Comment (Tonkpils) [reply lib/primer/classify/utilities.yml:1561]: Gotcha, so should I remove this then?

Comment (camertron) [reply lib/primer/classify/utilities.yml:1561]: Yes please!

Comment (Tonkpils) [reply lib/primer/classify/utilities.yml]: Done!

**Review by camertron — APPROVED** [review]

**Comment by camertron** [comment]
Shoot, sorry @Tonkpils! That change to utilities.yml was required to get tests to pass. I ran `rake utilities:build` which added it back in. Apologies for the noise.

**Pull request merged**